tests/Tests/400__ods_dashboard/420__model_serving/420__model_serving.robot

```diff
@@ -1,48 +1,82 @@
-
+${PRJ_TITLE}=    model-serving-project
+${PRJ_DESCRIPTION}=    project used for model serving tests
 
 *** Test Cases ***
-Validate Model Serving quickstart
-    [Documentation]    Test the quickstart of the model serving repo, Temporary until included in RHODS
-    [Tags]    ModelMesh_Serving
-    # TODO: Replace with http://robotframework.org/robotframework/latest/libraries/Process.html#Run%20Process
-    Run    git clone ${MS_REPO}
-    Run    cd modelmesh-serving/quickstart && ./quickstart.sh ${ODH_NAMESPACE} ${MODEL_MESH_NAMESPACE}
+# Validate Model Serving quickstart
+#     [Documentation]    Test the quickstart of the model serving repo, Temporary until included in RHODS
+#     [Tags]    ModelMesh_Serving
+#     # TODO: Replace with http://robotframework.org/robotframework/latest/libraries/Process.html#Run%20Process
+#     Run    git clone ${MS_REPO}
+#     Run    cd modelmesh-serving/quickstart && ./quickstart.sh ${ODH_NAMESPACE} ${MODEL_MESH_NAMESPACE}
 
 Verify Model Serving Installation
     [Documentation]    Verifies Model Serving resources
     [Tags]    ModelMesh_Serving
-    Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  5 min  10 sec  Verify Openvino Deployment
+    # Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  5 min  10 sec  Verify Openvino Deployment
     Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  5 min  10 sec  Verify odh-model-controller Deployment
     Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  5 min  10 sec  Verify ModelMesh Deployment
-    Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  5 min  10 sec  Verify Minio Deployment
-    Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  5 min  10 sec  Verify Serving Service
+    # Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  5 min  10 sec  Verify Minio Deployment
+    # Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  5 min  10 sec  Verify Serving Service
     Run Keyword And Continue On Failure  Wait Until Keyword Succeeds  5 min  10 sec  Verify Etcd Pod
 
+Verify Model Can Be Deployed Via UI
+    Open Model Serving Home Page
+    # Verify No Models Are Present
+    Click Button  Create server
+    # Verify moved to DSP page
+    Create Data Science Project    title=${PRJ_TITLE}    description=${PRJ_DESCRIPTION}
+    Create S3 Data Connection    project_title=${PRJ_TITLE}    dc_name=model-serving-connection
+    ...            aws_access_key=${S3.AWS_ACCESS_KEY_ID}    aws_secret_access=${S3.AWS_SECRET_ACCESS_KEY}
+    ...            aws_bucket_name=ods-ci-s3
+    Create Model Server
+    Open Model Serving Home Page
+    Serve Model    project_name=${PRJ_TITLE}    model_name=test-model    framework=onnx    existing_data_connection=${TRUE}
```

Comment:
Would it be possible/good idea to add some checks here to ensure the previous operations have created the necessary items/resources? 
Maybe also the Verify Openvino Deployment and Verify Serving Service keywords could be moved here

Reply:
Yes, seems like a good idea

Reply:
The openvino deployment gets scaled to 1 pod only after the first model is set up, so I've put those checks after the `Serve Model` keyword